dias/task.py

```diff
@@ -218,32 +260,38 @@ def cleanup(self, dir, max_size, check=False):
         return (total_data_size, disk_usage)
 
     def increment(self):
-        """Increment start_time by period"""
+        """Increment start_time by period."""
         self.start_time += self.period
 
     # Rich comparison
     def __eq__(self, other):
+        """Tell if the name of this task equals the name of another task."""
```

Comment:
It would WARN about the "Ordering" section, so I am using "Hint" for this (https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html?highlight=napoleon)

The first line in a method docstring usually has to be in imperative mood. But this passes, so I'll use what you suggest.
The __magic__ methods don't make it to the API reference at the moment. Do you want them to?